setup.cfg

```diff
@@ -1,10 +1,6 @@
 [bdist_wheel]
 universal = 1
 
-[flake8]
-max-line-length = 140
```

Comment:
Why this section was deleted?

Reply:
It was being used by flake8 instead of .flake8 from pre-commit-vauxoo, so when linting, it failed (for example on F401 which is ignored in .flake8 but not here)